RELEASE.md

```diff
@@ -0,0 +1,3 @@
+Release type: patch
+
+Update entity resolver exception handling to set the result to the original error instead of a `GraphQLError`, which obscured the original message and meta-fields.
```

Comment:
```suggestion
Update federation entity resolver exception handling to set the result to the original error instead of a `GraphQLError`, which obscured the original message and meta-fields.
```